What is the purpose of the pnpm alias?

### Current Behavior

- When using nx cli with pnpm to generate, the cli advises to use an alias specifically stating `create a shortcut such as: alias pnx="pnpm nx --"`
- After setting up this alias, nx commands are no longer recognized due to the `--`
- setting up the alias using just 'pnpm nx' works fine.

### Expected Behavior

- CLI suggestion should suggest `create a shortcut such as: alias pnx="pnpm nx"` or not provide an alias recommendation since it appears to work fine with only utilizing nx.

### Github Repo

_No response_

### Steps to Reproduce

1. Using a workspace preference with pnpm, run any nx generate command and view suggested alias.
2. Setup suggested alias `alias pnx="pnpm nx --"`
3. Use alias to generate, example `pnx generate @nrwl/angular:library-secondary-entry-point ....`
4. Notice it executes but does nothing if the command is invalid, but does throw an nx report error if the command is valid.


### Nx Report

```shell
ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command failed with ENOENT: nx-- generate @nrwl/angular:library-secondary-entry-point --library=library-name --name=name-of-
spawn nx-- ENOENT
```


### Failure Logs

_No response_

### Additional Information

_No response_

--- a/packages/nx/bin/nx.ts
+++ b/packages/nx/bin/nx.ts
@@ -64,24 +64,6 @@ if (
   if (localNx === resolveNx(null)) {
     initLocal(workspace);
   } else {
-    const packageManager = detectPackageManager();
-    if (packageManager === 'pnpm') {
-      const tip =
-        process.platform === 'win32'
-          ? 'doskey pnx=pnpm nx -- $*'
-          : `alias pnx="pnpm nx --"`;
-      output.warn({
-        title: `Running global Nx CLI with PNPM may have issues.`,
-        bodyLines: [
-          `Prefer to use "pnpm" (https://pnpm.io/cli/exec) to execute commands in this workspace.`,
-          `${chalk.reset.inverse.bold.cyan(
-            ' TIP '
-          )} create a shortcut such as: ${chalk.bold.white(tip)}`,
-          ``,
-        ],
-      });
-    }
-
     // Nx is being run from globally installed CLI - hand off to the local
     require(localNx);
   }

